src/elements/content-explorer/Content.js

```diff
@@ -54,6 +55,7 @@ type Props = {
 const Content = ({
     currentCollection,
     focusedRow,
+    gridColumnCount = 0,
```

Comment:
What would the `ItemGrid` display if `gridColumnCount` actually was 0? Should it default to 1 instead?